mne/channels/layout.py

```diff
@@ -777,9 +780,16 @@ def read_montage(kind, ch_names=None, path=None, scale=True):
     path : str | None
         The path of the folder containing the montage file. Defaults to the
         mne/channels/data/montages folder in your mne-python installation.
-    scale : bool
-        Apply useful scaling for out the box plotting using montage.pos.
-        Defaults to True.
+    unit : 'mm' | 'm' | None
+        Values supported are 'mm', which will scale points up by 1e3,
+        'm' which will scale point down by 1e3, or None.
+        Default is None.
```

Comment:
+1 we should just always do it in `m`, it's what we've tried to do in most places. The places where it uses `mm` I frequently make mistakes.